### PR TITLE
some clean-up

### LINKED ISSUE #64: Use FastBroadcast.jl, `@muladd` etc. consistently
Right now, we use these annotations/features in some places but not necessarily everywhere. It would be good for consistency (and maybe also for performance) to reconsider this choice.

### PR DESCRIPTION
**Pull request by ranocha**
Closes #64

### CODE REVIEW

File: src/PositiveIntegrators.jl [hunk 1]
```diff
@@ -6,7 +6,6 @@ using SparseArrays: SparseArrays, AbstractSparseMatrix
 using StaticArrays: SVector, MVector, SMatrix, StaticArray, @SVector, @SMatrix
 
 using FastBroadcast: @..
-using Kwonly: @add_kwonly
```
Comment (ranocha) [inline src/PositiveIntegrators.jl:9]: We did not use this anywhere

**Review by SKopecz — CHANGES_REQUESTED** [review]
Thanks! Please `format` the code.

**Comment by SKopecz** [comment]
Thanks!

**Review by SKopecz — COMMENTED** [review]
Thanks!

**Pull request merged**